webrender/src/batch.rs

```diff
@@ -211,102 +369,116 @@ impl AlphaBatchList {
         // The bounding box of everything at this Z plane. We expect potentially
         // multiple primitive segments coming with the same `z_id`.
         z_bounding_rect: &PictureRect,
-        z_id: ZBufferId,
+        _z_id: ZBufferId,
```

Comment:
I guess we don't need this check anymore, but let's remove the param altogether if that's the case?